src/app/components/search/search.js

```diff
@@ -90,7 +90,7 @@ angular
 
             var watchExpressions = ['query', 'checkboxStatus.show_names', 'checkboxStatus.show_descriptions', 'checkboxStatus.show_columns', 'checkboxStatus.show_column_descriptions', 'checkboxStatus.show_code', 'checkboxStatus.show_tags'];
             scope.$watchGroup(watchExpressions, function() {
-                scope.results = filterResults(projectService.search(scope.query), scope.checkboxStatus);
+                scope.filteredResults = filterResults(scope.results, scope.checkboxStatus);
```

Comment:
For what reason did you change `projectService.search(scope.query)` to `scope.results`?

Reply:
This was the best comment ever! I re-structured my changes and now it aligns with the project.